inc/class-upgrade.php

```diff
@@ -899,4 +900,14 @@ private function migrate_woocommerce_archive_setting_to_shop_page() {
 			WPSEO_Options::set( 'noindex-ptarchive-product', false );
 		}
 	}
+
+	/**
+	 * Performs the 14.4 upgrade for admin redesign.
+	 *
+	 * Removes unused database usermeta.
+	 */
+	private function upgrade_144() {
+		global $wpdb;
+		$wpdb->query( "DELETE FROM $wpdb->usermeta WHERE meta_key = 'wpseo-dismiss-configuration-notice'" );
+	}
```

Comment:
I think in this file all the `upgrade_*` methods should be grouped together and utility methods should stay at the end of the file.